### ISSUE TITLE
Can we have a "logged_metric" type 

### ISSUE THREAD

**Issue by supriyopaul** [deep-compute/basescript#71]
If we specify the type as "metric" basescript does grouping and also drops the "__fields", I want something wich will mention the type as "metric" and but not drop  the "__fields".

**Comment by prashanthellina** [comment]
@Rajendra-R  I recommend we support "logged_metric" in addition to "metric" and "log" for `type` parameter. When user specifies "logged_metric", we can disable grouping behavior for that alone and instead directly write to output with `type=metric`.

**Comment by ghost** [comment]
Sure @prashanthellina We will support that.

**Comment by ghost** [comment]
@prashanthellina 
* As per our requirement we implemented the feature for customized stop metric grouping by specifing `type="logged_metric"`.
#### Sample-code:
```python
from basescript import BaseScript
import time
import random
import uuid

class Stats(BaseScript):
    def __init__(self):
        super(Stats, self).__init__()

    def run(self):
        ts = time.time()
        while True:
            # New Metric Format.
            self.log.info("server_stats", company="deepcompute", user="rajendra", _uid=uuid.uuid1().hex,
                time_duration=(time.time()-ts), type="metric", random_number=random.randint(1, 50))
            # New metric tag for not grouping
            self.log.info("test new metric", _uid=uuid.uuid1().hex, user="rajendra", type="logged_metric")

if __name__ == '__main__':
    Stats().start()
```
#### output:
![screen shot 2018-06-12 at 6 16 36 pm](https://user-images.githubusercontent.com/16955073/41291458-5fe6c65a-6e6d-11e8-9ae7-e1a2748959e9.png)

**Comment by prashanthellina** [comment]
@Rajendra-R  Thank you!

**Comment by ghost** [comment]
@supriyopaul We resolved this issue in new release = 0.2.6 so we can close this ticket.
